core/web/presenters/ocr_keys.go

```diff
@@ -57,9 +58,9 @@ func NewOCR2KeysBundleResource(key ocr2key.KeyBundle) *OCR2KeysBundleResource {
 	return &OCR2KeysBundleResource{
 		JAID:              NewJAID(key.ID()),
 		ChainType:         string(key.ChainType()),
-		OnchainPublicKey:  key.OnChainPublicKey(),
-		OffChainPublicKey: hex.EncodeToString(key.OffchainPublicKey()),
-		ConfigPublicKey:   hex.EncodeToString(configPublic[:]),
+		OnchainPublicKey:  fmt.Sprintf("ocr2on_%s", key.OnChainPublicKey()),
```

Comment:
The prefix here should include some information about _what_ chain is being targeted, since we will use different cryptographic schemes (and thus keys) for different chains